### PR TITLE
 BUGFIX: ISSUE-001 added some details to readme

### LINKED ISSUE #1: missing readme details
it's missing the readme details.. oh no..

### PR DESCRIPTION
**Pull request by yoinasen**
issue: 001 missing readme details
...

### CODE REVIEW

**Review by yoavshtainer — COMMENTED** [review]
How you dare???

**Review by yoavshtainer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# integrator_test
\ No newline at end of file
+# integrator_test
+
```
Comment (linafeldman) [inline README.md:2]: test

**Review by linafeldman — APPROVED** [review]

**Review by AlonRiani — APPROVED** [review]

**Review by linafeldman — APPROVED** [review]

**Review by linafeldman — APPROVED** [review]

**Review by linafeldman — APPROVED** [review]

**Review by linafeldman — APPROVED** [review]
test2

**Review by linafeldman — APPROVED** [review]
test3

**Pull request merged**